addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -310,6 +310,19 @@ export class Rtc extends Record {
                 this.sfuClient?.disconnect();
             }
         });
+        browser.navigator.serviceWorker?.addEventListener("message", ({ data: { action, id } }) => {
+            if (action === "join_call") {
+                const channel = this.store.Thread.get({ model: "discuss.channel", id });
+                if (!channel) {
+                    return;
+                }
+                channel.open();
+                if (this.state.channel) {
+                    return;
+                }
+                this.toggleCall(channel);
```

Comment:
Toggle methods are really not a great api, as they are confusing and prone to errors.

If possible now, or on another PR, this should be removed and immediately call "join" instead.